packages/api-headless-cms/src/crud/contentEntry.crud.ts

```diff
@@ -189,7 +189,7 @@ interface DeleteEntryParams {
 const createEntryId = (input: CreateCmsEntryInput) => {
     let entryId = mdbid();
     if (input.id) {
-        if (input.id.match(/^([a-zA-Z0-9])([a-zA-Z0-9\-]+)([a-zA-Z0-9])$/) === null) {
+        if (input.id.match(/^([a-zA-Z0-9])([a-zA-Z0-9\-#]+)([a-zA-Z0-9])$/) === null) {
             throw new WebinyError(
```

Comment:
We have added `#` because `createEntryId` did not support ids with `#`, and in our case form id contains `#`.